handlers/api-handler.ts

```diff
@@ -169,18 +179,31 @@ export function ApiHandler<
 				}
 			}
 
-			const validateBodyResult = validateInput(
-				ajvValidators.body,
-				event.body ?? {},
-			);
+			let validateBodyResult: ValidationResult<B> = {
+				success: true,
+				data: event.body as unknown as B,
+			};
+			let validateQueryResult: ValidationResult<Q> = {
+				success: true,
+				data: queryBody as Q,
+			};
+			if (validators) {
+				validateBodyResult = validateInput(validators.body, event.body);
+				validateQueryResult = validateInput(validators.query, queryBody);
```

Comment:
The intention here is to give `validators` top priority if both validation methods are specified.